modules/storage-persistent-storage-iscsi-multipath.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage-iscsi.adoc
+
+[[iscsi-multipath]]
+= iSCSI Multipathing
+For iSCSI-based storage, you can configure multiple paths by using the 
+same IQN for more than one target portal IP address. Multipathing ensures 
+access to the persistent volume when one or more of the components in a 
+path fail.
+
+To specify multi-paths in pod specification use the `portals` field. 
```

Comment:
s/in pod specification/in the pod specification

Reply:
Corrected this sentence.